Chimoney/chimoney-community-projects
# Chimoney SlackBot

## Pull Request (PR) Note

### Description
This PR introduces the `/sendchimoney` command to our Slack app. This command allows users to send Chimoney to other users within the workspace. It enhances our app's functionality by facilitating payments and interactions between users.

### Testing Instructions
To test the new `/sendchimoney` command, follow these steps:

1. Join our Slack workspace for testing: [Test Workspace](https://join.slack.com/t/zee-p4j1547/shared_invite/zt-260v2mkck-EX~V4GasNEvjjH3c6aPkBw) {Recommended for now}

2. Install the app to the workspace using the following link: [Install App](https://stingray-app-3r8tj.ondigitalocean.app/slack/install)

3. Once the app is installed, open a direct message or a channel within the workspace.

4. Type `/sendchimoney` followed by the desired amount and the usernames of the recipients. For example:
   - `/sendchimoney $50 @user123 @user456` sends $50 in Chimoney to users @user123 and @user456.

5. Verify that the command executes successfully, and the recipients receive the payment link.

### Additional Notes
- Ensure that the Slack app is installed in the workspace before testing.
- If you encounter any issues during testing or have feedback, please report them in this PR.

Happy testing!
@uchibeke The slackbot
